lib/samples/display_clusters/README.md

```diff
@@ -0,0 +1,39 @@
+# Display clusters
+
+Display a web map with a point feature layer that has feature reduction enabled to aggregate points into clusters.
+
+![Image of display clusters](display_clusters.png)
+
+## Use case
+
+Feature clustering can be used to dynamically aggregate groups of points that are within proximity of each other in order to represent each group with a single symbol. Such grouping allows you to see patterns in the data that are difficult to visualize when a layer contains hundreds or thousands of points that overlap and cover each other.
+
+## How to use the sample
+
+Pan and zoom the map to view how clustering is dynamically updated. Toggle clustering off to view the original point features that make up the clustered elements. When clustering is On, you can click on a clustered geoelement to view aggregated information and summary statistics for that cluster as well as a list of containing geo elements. When clustering is disabled and you click on the original feature you get access to information about individual power plant features.
+
+## How it works
+
+1. Create a map from a web map. (PortalItem reference in common design README doesn't apply?)
```

Comment:
PortalItem reference in common design README doesn't apply here right?
@jenmerritt